CorpusSearch/ClientApp/src/routes/BitPlayer.tsx

```diff
@@ -63,23 +63,72 @@ function Video({vId="rLEBp8R1_XA"}) {
         setKtime(r_player.current.getCurrentTime())
         restart()
     }
-
     /* eslint-enable */
+    const alterTime = (timeString : string,timeFunc :React.Dispatch<React.SetStateAction<string>> ,chg : number) => {
+        const time_val = parseFloat(timeString) + chg
```

Comment:
It would make sense to see if you can make `startTime` a number, and have a separate representation as the textbox value. 

TimeString is not guaranteed to be a number

Reply:
It. sort of is though. the input has type="number" so you can't put in alphas that are not numbers. You can put in weird numbers though  like 011110. Might be able to add a regex pattern to restrict further. Or use something like https://www.npmjs.com/package/react-number-format

Reply:
Hmm. I saw [this discussion](https://mui.com/material-ui/react-text-field/) after reading [stackoverflow](https://stackoverflow.com/questions/43067719/how-to-allow-only-numbers-in-textbox-in-reactjs) which recommended https://mui.com/base-ui/react-number-input/